Makefile.am

```diff
@@ -43,9 +43,7 @@ AM_CFLAGS = @STRICT_CFLAGS@
 # Make sure to run Gtk-doc tests and build the documentation when doing
 # 'make distcheck'
 AM_DISTCHECK_CONFIGURE_FLAGS = \
```

Comment:
Still need to update https://github.com/endlessm/eos-metrics/blob/master/Makefile.am#L64.